sidecar/proxy/envoy/manager.go

```diff
@@ -485,6 +484,12 @@ func sanitizeRules(ruleList []api.Rule) {
 					}
 				}
 			}
+
+			sum = 0
+			for j := range rule.Route.Backends {
+				sum += rule.Route.Backends[j].Weight
+				rule.Route.Backends[j].Weight = sum
+			}
```

Comment:
We don't need this with  weighted_clusters right? The weights there are absolute weights.